packages/menu/test/menu-group.test.ts

```diff
@@ -12,14 +12,20 @@ governing permissions and limitations under the License.
 import '../sp-menu-group.js';
 import '../sp-menu.js';
 import '../sp-menu-item.js';
-import { Menu } from '../';
-import { fixture, elementUpdated, html, expect } from '@open-wc/testing';
+import { Menu, MenuItem, MenuGroup } from '../';
+import {
+    fixture,
+    elementUpdated,
+    html,
+    expect,
+    waitUntil,
+} from '@open-wc/testing';
 
 describe('Menu group', () => {
     it('renders', async () => {
         const el = await fixture<Menu>(
             html`
-                <sp-menu>
+                <sp-menu selects="none">
```

Comment:
Right now this outer menu won't manage the child `sp-menu-group` elements without selects being one of `none`, `single`, or `multiple`, but I'm thinking it might be better for the behavior to default to `selects="none"` for the outer parent menu (i.e. if there's nothing to inherit from the parent).

Reply:
I also realize that we need a way to have an "unmanaged" state for the `PickerBase` case, since at least for this round `PickerBase` is still managing it's own selection.

Reply:
Sub-note, styling for the `Menu` in a `Picker` will currently require a `selects` attribute (value not withstanding) in order to display correctly when going between having a selected child and not. That also means that if `<sp-menu selects="none">` actually doesn't take a selected child that it will be visual "incorrect".

This likely doesn't help. Can you clarify what information would be good for me to share my thoughts on here?

Reply:
Oh yeah -- I even looked at that styling earlier today. I think I also need to fix styling for nested menus so I'll leave this open until that's resolved.

It'd be good to get your thoughts on having the default behavior be that selection is unmanaged (if no parent is managing selection). I think this also means that by default the styling should allow for selection. As I think about it more it seems reasonable, but we could certainly go a different direction.

Reply:
I don't understand this. Can you share a couple of code samples and the expected output?

Reply:
It's probably easiest to think about the difference between `selects="none"` and no selects.

For `selects="none"` we don't allow selection (and in the menu I'm capturing the click event, though that might be unnecessary now). This means that clients should add click events to the individual menu items here, at least the way it's coded now. When adding/updating items we also might clear the selected state on any items marked as selected.

When `selects` isn't present, selection is totally unmanaged. Click events are passed up so either a parent menu or PickerBase can handle it in the way they need. In general this probably isn't the state you want if there's not a parent managing things.

The one piece of this I haven't thought through is whether the default styling with no `selects` should be that the items have the selectable styling or not. Keeping it selectable is handy for the PickerBase case.